authentication/auth_method_oidc.go

```diff
@@ -65,19 +68,39 @@ func (a oidcAuth) getMSALToken(ctx context.Context, api environments.Api, _ auto
 		return &authWrapper.Authorizer{Authorizer: conf.TokenSource(ctx)}, nil
 	}
 
+	idToken := a.idToken
+
+	if a.idTokenPath != "" {
```

Comment:
Not sure on priority of the token path vs actual token here, or if we would want to error out when both are supplied instead.